### PR TITLE
2023-06-07-NER__DemographicExtended_Clinical_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/andrei9825/2023-06-07-NER__DemographicExtended_Clinical_en.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+layout: model
+title: Demographic_Extended
+author: John Snow Labs
+name: NER__DemographicExtended_Clinical
+date: 2023-06-07
+tags: [en, licensed]
+task: Named Entity Recognition
+language: en
+edition: Healthcare NLP 4.3.1
```
Comment (Cabir40) [inline docs/_posts/andrei9825/2023-06-07-NER__DemographicExtended_Clinical_en.md:10]: you can upload with 4.4.3

File: docs/_posts/andrei9825/2023-06-07-NER__DemographicExtended_Clinical_en.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+word_embeddings = WordEmbeddingsModel.pretrained("embeddings_healthcare_100d", "en", "clinical/models")\
+    .setInputCols(["sentence", "token"])\
+    .setOutputCol("embeddings")
+
+ner = MedicalNerModel.pretrained("NER__DemographicExtended_Clinical","en","clinical/models")\
+    .setInputCols(["sentence","token","embeddings"])\
+    .setOutputCol("ner")\
+    .setLabelCasing("upper")
+    
+ner_converter = NerConverter() \
```
Comment (Cabir40) [inline docs/_posts/andrei9825/2023-06-07-NER__DemographicExtended_Clinical_en.md:61]: please use `NerConverterInternal`

File: docs/_posts/andrei9825/2023-06-07-NER__DemographicExtended_Clinical_en.md (hunk 1 above)
Comment (andrei9825) [reply docs/_posts/andrei9825/2023-06-07-NER__DemographicExtended_Clinical_en.md:10]: I can only select the 4.4.2 from the dropwdown

**Comment by Cabir40** [comment]
will be re-upload

**Pull request closed without merging**